src/dockerflow/sanic/checks.py

```diff
@@ -54,12 +54,9 @@ async def check_redis_connected(redis):
         dockerflow = Dockerflow(app, redis=redis)
 
     """
-    import aioredis
+    from redis import asyncio as aioredis
 
-    if aioredis.__version__.startswith("1."):
-        RedisConnectionError = aioredis.ConnectionClosedError
-    else:
-        RedisConnectionError = aioredis.ConnectionError
+    RedisConnectionError = aioredis.ConnectionError
```

Comment:
nit: could be further simplified, something like:

```python
from redis.asyncio import ConnectionError as RedisConnectionError
```

or even change `RedisConnectionError` in the except block below:

```python
    except ConnectionError as e:
        ...
 ```

Reply:
[b850e82](https://github.com/mozilla-services/python-dockerflow/pull/76/commits/b850e8202bf7a7ff15ff63b4b2f6199caedec3c2)